docs/running-on-yarn.md

```diff
@@ -54,8 +54,8 @@ In `yarn-cluster` mode, the driver runs on a different machine than the client,
 
 # Preparations
 
-Running Spark-on-YARN requires a binary distribution of Spark which is built with YARN support.
-Binary distributions can be downloaded from the Spark project website. 
+Running Spark on YARN requires a binary distribution of Spark which is built with YARN support.
```

Comment:
What about lines 30 and 24. They were the most important with the others just amendments as I was reviewing the entire doc. The reason for the change was this change 16b6d18613e150c7038c613992d80a7828413e66 where I learnt the command line's option is no longer supported and hence the change in the doc.